nullaway/src/main/java/com/uber/nullaway/handlers/RxNullabilityPropagator.java

```diff
@@ -367,4 +427,189 @@ public void onDataflowVisitReturn(
             }
         }
     }
+
```

Comment:
javadoc the class declaration at least?

Reply:
Will do. These classes below are considered internal implementation details of `RxNullabilityPropagator`, hence the private scope, but will add a javadoc for the classes as documentation for this file :)